brunofurmon/mayt-api
# Feature/docker

# What this PR does?

-   Adds docker integration
-   Prettier logs

## - Before/After

-   Logs were plain. Now they have color for each criticity and an UTC timestamp.

## - How to test it?

- Make sure `docker` is running.
- `make setup && make start` should listen to port 3001

## - TODOs

-   None

## - Risks

-   None
